README.rst

```diff
@@ -36,6 +36,9 @@ Installation
 umpf is a bash script, so no installation is necessary. It just needs a few
 command line tools such as sed, grep and of course git.
 
+To enable bash completion for umpf source .../umpf/bash_completion in
+~/.bash_completion
```

Comment:
I think it's better to recommend [on-demand/dynamic completion loading](https://github.com/scop/bash-completion/blob/master/README.md):
```suggestion
To enable bash completion, make sure umpf is in your ``$PATH``, then::
    $ mkdir -p ~/.local/share/bash-completion/completions
    $ ln -s /path/to/umpf/bash_completion ~/.local/share/bash-completion/completions/umpf
```

Reply:
I didn't know about that. Fixed.